editor/components/post-title/index.js

```diff
@@ -116,6 +115,7 @@ class PostTitle extends Component {
 							onKeyPress={ this.onUnselect }
 						/>
 					</KeyboardShortcuts>
+					{ isSelected && <PostPermalink /> }
```

Comment:
Is there a reason this was moved?

Reply:
Yup, so it comes after the `PostTitle` in the tab order.